content/en/synthetics/private_locations/dimensioning.md

```diff
@@ -16,6 +16,14 @@ By defining a test type, maximum number of test runs, and total hardware require
 
 To improve dimensioning, split your test assignments based on test types. For example, you can have some private locations run only API and multistep API tests while other private locations run only browser tests.
 
+### Prerequisites
+
+You will need:
```

Comment:
@m-rousse @troisdiz can I get you both to review this when you have time? 🙏🏻

Reply:
```suggestion
To get started with dimensioning your private locations, you need the following:
```